mlflow/genai/optimize/optimizers/gepa_optimizer.py

```diff
@@ -30,6 +30,33 @@ class GepaPromptOptimizer(BasePromptOptimizer):
             Default: 100
         display_progress_bar: Whether to show a progress bar during optimization.
             Default: False
+        gepa_kwargs: (Optional) Additional keyword arguments to pass directly to
+           [gepa.optimize](https://github.com/gepa-ai/gepa/blob/main/src/gepa/api.py).
+            Useful for accessing advanced GEPA features not directly exposed
+            through MLFlow's GEPA interface.
+
+            Available parameters:
```

Comment:
Do we need to list all gepa parameters? I think we can just refer to gepa documentation